<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Warhouses</class>
 <widget class="QFrame" name="F_Warhouses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="L_info">
     <property name="text">
      <string>Warehouses</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutTop">
     <item>
      <widget class="QPushButton" name="Bt_AddNew">
       <property name="text">
        <string>New Warehouse +</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="F_AddNew">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QSpinBox" name="Sb_ID">
          <property name="minimum">
           <number>-2</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="Le_WarehouseName">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>Warehouse Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Le_Description">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>Measurements</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Cb_Status">
            <item>
             <property name="text">
              <string>Active</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Delete</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Closed</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bt_save">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bt_cancel">
            <property name="text">
             <string>Cancel</string>
            </property>
            <property name="shortcut">
             <string>Esc</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutMiddle">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_infoTable">
       <property name="text">
        <string>1 to 10 elements from 400</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBottom">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutControle">
       <item>
        <widget class="QPushButton" name="Bt_Next">
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_Pages"/>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_previous">
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
